cmd/version.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+// versionCmd represents the version command
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Get binary version.",
+	Long:  `Command to output version of a binary.`,
+	Run:   getVersion,
+}
+
+// Version string populated by releaser Job
+var (
+	Version string = "0.0.0"
+	// commitHash contains the current Git revision.
+	CommitHash string
+	// Timestamp contains the UnixTimestamp of the binary build.
+	Timestamp string
+	// VendorInfo contains vendor notes about the current build.
+	VendorInfo string
```

Comment:
Can you keep default values over here? so that it can display the default values when someone builds directly using `go build` without makefile?

Reply:
Ack, 
Added default values in version.go, Kept Build time variables in Makefile only